tests/unit/test_models/test_full_battery_models/test_lithium_ion/test_spm.py

```diff
@@ -1,27 +1,26 @@
 #
 # Tests for the lithium-ion SPM model
 #
-from tests import TestCase
 import pybamm
-import unittest
 from tests import BaseUnitTestLithiumIon
+import pytest
 
 
-class TestSPM(BaseUnitTestLithiumIon, TestCase):
-    def setUp(self):
+class TestSPM(BaseUnitTestLithiumIon):
+    def setup_method(self):
```

Comment:
```suggestion
    @pytest.fixture(autouse=True)
    def setUp(self):
```

I think you missed adding the fixture here :) and let's keep the name of the function the same as other files for consistency.

Reply:
Ah I get it. pytestify changes them to pytest equivalent of `setUp` https://docs.pytest.org/en/6.2.x/xunit_setup.html#:~:text=Similarly%2C%20the%20following%20methods%20are%20called%20around%20each%20method%20invocation%3A
This can also be achieved via fixtures which we have already done before. But yeah we should stick to one way.

Reply:
Let's replace the `setUp` methods everywhere with these `pytest` equivalents that you've mentioned – in another PR though, not here.